tests/acceptance/run.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
I think you are now calling the core ``tests/acceptance/run.sh`` from ``make test-acceptance-cli`` in ``Makefile``
So this ``run.sh`` is not used and can be deleted?

Reply:
:+1: